### DISCUSSION TITLE
球场真人 | /cn/2019/07/badminton-mind/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1341] [Comments]
https://yihui.org/cn/2019/07/badminton-mind/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
正是这样。只不过目前还抽不远，抽回去通常下一拍就被人挡在网前了，我来不及转身跑回网前。

https://user-images.githubusercontent.com/163582/267533028-e6f5279b-3ac4-457b-90e7-62b9ea22dfaa.mp4

这种球我印象最深的是下面这个（李宗伟都被打蒙了）：

<https://www.youtube.com/watch?v=V-AoOB0qqLQ>

> ——原帖发布于 2019-08-05 03:19:33

**Comment by residualsun1** [comment]
益辉，祝你生日快乐，身体健康！在你的网站里溜达了一会儿，发现最近一年能对应上你生日的正好是这一篇。很高兴可以认识你，因为用你的 R 包和看你的博客，我也开始写博客，写到现在也有 2 年了，未来也还会一直写下去（ 顺便来催一催更！

**Reply by yihui** [reply]
多谢你啊！暑假里我回国了一趟，Github 消息堆积成山，近日终于等到了全体神兽回笼这才开始处理。

在这个生日主题下，结合刚快速浏览的你的近况，我献丑送你一首歌吧：https://kg.qq.com/node/play?s=Qq3Z312QkNkR42Ap （粤语基本上是我对着拼音唱的，还望别笑太大声——我是多么羡慕会讲粤语的朋友们啊）我个人实在是太喜欢王菲这首歌。希望你可以找到你远远的星宿。

又及：以前我看你有个唱歌的页面，上个月好像给拆了。有朋踏歌而行，不亦乐乎？

再及：催更催得很好，下次接着催。

**Reply by residualsun1** [reply]
不是献丑！但我也不跟你客气，直接「收下」你唱的歌 :D

我不懂啥节奏和乐理，一直都是认感觉的，自己唱的时候也都是跟着感觉走，但老歌好听或许正是因为难以言喻的感觉吧！王菲唱的《但愿人长久》、邓丽君唱的《我只在乎你》等等，我听的都是一种独一无二的感觉，或许你的粤语没有 200% 标准（怎么说这也是我唯一的优势了阿喂），但唱的正是「好听」不该缺少的一种独特和真实的感觉。

> 又及：以前我看你有个唱歌的页面，上个月好像给拆了。有朋踏歌而行，不亦乐乎？

那个页面太简陋了，等我以后攒一些库存再翻新一番 : )！

> 再及：催更催得很好，下次接着催。

基本可以确认下次更新的动静有多大了。
